docs/modules/readable-stream.md

```diff
@@ -0,0 +1,9 @@
+# readable-stream
+
+[`readable-stream`](https://www.npmjs.com/package/readable-stream) is a mirror of the NodeJS `stream` module.
+
+# Alternatives
+
+## NodeJS (since v0.9.4)
+
+If your NodeJS version is recent it is better to directly use [`stream`](https://nodejs.org/api/stream.html).
```

Comment:
Maybe makes sense to mention streams from web standards, like https://developer.mozilla.org/en-US/docs/Web/API/Streams_API ?

Reply:
It could, but they aren't a direct alternative to NodeJS streams, they need to be converted from one to the other, no?

Still, I can add another alternative

Reply:
Maybe. I just don't know is `readable-stream` NodeJS-only package or not. If yes - it's not required, if not - I think that it should be mentioned.

Reply:
> v4.x.x of readable-stream is a cut from Node 18. This version supports Node 12, 14, 16 and 18, **as well as evergreen browsers**

Reply:
ah yes indeed, I added an alternative for Streams API